api/src/environment.js

```diff
@@ -6,7 +6,8 @@ const { UNIT_TEST_ENV, COUCH_URL, MEDIC_API_RESOURCE_PATH, NODE_ENV } = process.
 
 if (UNIT_TEST_ENV) {
   module.exports = {
-    serverUrl: ''
+    serverUrl: 'ftp://myserver',
+    db: 'mydb'
```

Comment:
I'm not sure this change is needed.

Reply:
I think you're right now. I was trying something then I backed out of it. I'll revert and make sure everything still passes.